### PR TITLE
What to do on a rainy day ☔️

### PR DESCRIPTION
**Pull request by Snabbina**
My reflections on how this week's project turned out:
The det up and all the dependencies was a project on its own, otherwise cool to make a working mobile app in just a few days!

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import styled from 'styled-components/native'
+import { ThingsToDo } from './components/ThingsToDo'
```
Comment (EmOlsz) [inline App.js:3]: Let's not forget about one empty line between outer and local dependencies!

File: components/ThingsToDo.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import styled from "styled-components/native";
+import { Text, View, Vibration, TouchableOpacity } from 'react-native';
```
Comment (EmOlsz) [inline components/ThingsToDo.js:3]: I see that sometimes you use single quotes and sometimes double quotes - let's try to be consistent about it

File: components/ThingsToDo.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+const ThingsToDoText = styled.Text`
+  font-size: 20px;
+  text-align: center;
+  color: white;
+`;
+
+export const ThingsToDo = () => {
+  // array of objects of things to do
+
+  const ThingsToDoArray = [
```
Comment (EmOlsz) [inline components/ThingsToDo.js:44]: Uppercase names are reserved for components only, not for arrays

File: components/ThingsToDo.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      {
+        thingtodo: "Ha på pyamas hela dagen!",
+        color: "#e2dcd4", //grey
+        emoji: "",
+      },
+  ];
+
+  // An randommaking function that gets invoked when button gets push.
+
+    const [thingtodo, setThingToDo] = useState({});
```
Comment (EmOlsz) [inline components/ThingsToDo.js:74]: Wrong indentation over here ;)

File: components/ThingsToDo.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+  const getThingsToDo = () => {
+    const theThingToDo =
+      ThingsToDoArray[Math.floor(Math.random() * ThingsToDoArray.length)];
+    setThingToDo(theThingToDo);
+  };
+
+  return (
+    <Container>
+      <ThingsToDoContainer style={{ backgroundColor: thingtodo.color }}>
```
Comment (EmOlsz) [inline components/ThingsToDo.js:84]: If we decide to use styled components, then let's stick to it and do not pass styles via `style` attribute.
You can pass `thingtodo` value as prop to your styled component and later on use it there to define styles

File: components/ThingsToDo.js [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+        <ThingsToDoText>{thingtodo.thingtodo}</ThingsToDoText>
+        <ThingsToDoText>{thingtodo.emoji}</ThingsToDoText>
+      </ThingsToDoContainer>
+      <PickButton
+        onPress={() => {
+          getThingsToDo();
+          Vibration.vibrate();
```
Comment (EmOlsz) [inline components/ThingsToDo.js:91]: If we need to execute more than one line of code, then we should move our logic to separate function and call it on `onPress` :

```
const myCustomFunc = () => {
  getThingsToDo();
  Vibration.vibrate();
}

return (
  <PickButton onPress={myCustomFunc} />
)
```

**Review by EmOlsz — CHANGES_REQUESTED** [review]

**Comment by Snabbina** [comment]
Hello, made the changes requested on this project, is it ok now? /S

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**